.eslintrc.js

```diff
@@ -3,11 +3,16 @@ module.exports = {
     es6: true,
     node: true,
   },
-  rules:{
-    "prettier/prettier" : "error",
-    "eqeqeq": "error"
+  rules: {
+    "prettier/prettier": "error",
+    eqeqeq: "error",
   },
-  extends: ["prettier"],
+  extends: [
+    "prettier",
+    "eslint:recommended",
```

Comment:
should we stick to just `prettier` or apply more rules from 
- eslint-recommended
- typescript-eslint/recommended

Reply:
You can add them before `prettier`, the bottom rules will override the upper one AFAK.